test/controllers/trips_controller_test.rb

```diff
@@ -1,23 +1,108 @@
 require "test_helper"
 
 describe TripsController do
+  let (:trip) {
+    Trip.create(
+      driver_id: Driver.create(name: "sample name", vin: "12345678ABCDO").id,
+      passenger_id: Passenger.create(name: "another name", phone_num: "415-555-5555").id,
+      date: "12/24/19",
+      cost: 0,
+    )
+  }
+
   describe "show" do
-    # Your tests go here
+    it "can get a valid trip" do
+      get trip_path(trip.id)
+      must_respond_with :success
+    end
+
+    it "will show a 404 error for an invalid trip" do
+      get trip_path(-1)
+
+      must_respond_with :not_found
+    end
+  end
+
+  describe "create" do
+    it "can create a new trip" do
```

Comment:
You don't create a trip with a form like this.  Instead we just pass in a route parameter with the passenger_id.